### ISSUE TITLE
make chat where we can discuss about BNM

### ISSUE THREAD

**Issue by Lsc0x80** [ByNameModding/BNM-Android#49]
Сделай чат где мы сможем задавать вопросы друг другу, в тг, дискорде, где пожелаешь вообщем
Make a chat where we can ask each other questions, in telegram, discord, wherever you want in general

**Comment by yujack008** [comment]
good idea

**Comment by KingModder92** [comment]
Does bnm work for 64bit , arm64-v8a?

**Comment by YummyBacon5** [comment]
> Does bnm work for 64bit , arm64-v8a?

It should, as long have you have the correct hooking libraries and compile for the correct version, etc

**Comment by chinhtam97** [comment]
I am making a mod menu using zygisk. Previously I was using an old version that only supported armeabi-v7a.
I used Il2cppAttach before GetMethodOffset. So with the current version I can use it anymore? Please give me an example

**Comment by emrtyns** [comment]
My friend, how can we add the dll to the hooking method with names? I need to hook assembly.dll.

**Comment by XtraCube** [comment]
I agree with this, we need some community chat so people can support each other

**Comment by ThatOneGuyHeh** [comment]
> My friend, how can we add the dll to the hooking method with names? I need to hook assembly.dll.

You can use Image, or hooking this way:

```
InvokeHook(Class(BNM_OBFUSCATE("Namespace"), BNM_OBFUSCATE("Class"), Image(
            BNM_OBFUSCATE("Assembly-CSharp.dll"))).GetMethod(BNM_OBFUSCATE("Method")), new_HookedMethod, OldMethod);
```
